[2021-05-25 21:22:50,923] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T20:39:44.245454+00:00 [queued]>
[2021-05-25 21:22:50,997] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T20:39:44.245454+00:00 [queued]>
[2021-05-25 21:22:50,998] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 21:22:50,999] {taskinstance.py:1043} INFO - Starting attempt 2 of 2
[2021-05-25 21:22:51,000] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-05-25 21:22:51,049] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): Get_COVID19_Cases_For_Jordan_2020ToNow> on 2021-05-25T20:39:44.245454+00:00
[2021-05-25 21:22:51,075] {standard_task_runner.py:52} INFO - Started process 273 to run task
[2021-05-25 21:22:51,107] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'covid19_cases_Jordan', 'Get_COVID19_Cases_For_Jordan_2020ToNow', '2021-05-25T20:39:44.245454+00:00', '--job-id', '121', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/workflow.py', '--cfg-path', '/tmp/tmppoea95u8', '--error-file', '/tmp/tmplyen1x5y']
[2021-05-25 21:22:51,128] {standard_task_runner.py:77} INFO - Job 121: Subtask Get_COVID19_Cases_For_Jordan_2020ToNow
[2021-05-25 21:22:51,368] {logging_mixin.py:104} INFO - Running <TaskInstance: covid19_cases_Jordan.Get_COVID19_Cases_For_Jordan_2020ToNow 2021-05-25T20:39:44.245454+00:00 [running]> on host 4e0af5d83701
[2021-05-25 21:22:51,521] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=waed
AIRFLOW_CTX_DAG_ID=covid19_cases_Jordan
AIRFLOW_CTX_TASK_ID=Get_COVID19_Cases_For_Jordan_2020ToNow
AIRFLOW_CTX_EXECUTION_DATE=2021-05-25T20:39:44.245454+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-05-25T20:39:44.245454+00:00
[2021-05-25 21:38:08,895] {local_task_job.py:77} ERROR - Received SIGTERM. Terminating subprocesses
[2021-05-25 21:38:09,088] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 273
[2021-05-25 21:38:09,089] {taskinstance.py:1239} ERROR - Received SIGTERM. Terminating subprocesses.
